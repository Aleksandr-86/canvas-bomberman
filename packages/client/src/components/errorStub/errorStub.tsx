import { HeroBomb } from '../heroBomb/heroBomb'

import styles from './errorStub.module.css'

export const ErrorStub = () => {
  return (
    <div className={styles.errorStub}>
      <p className={styles.errorStubText}>
        <span className={styles.errorStubTextDanger}>БАААБАААХ!</span> И ВСЁ
        СЛОМАЛОСЬ!
      </p>
      <HeroBomb />
      <p className={styles.errorStubText}>НО МЫ УЖЕ ЧИНИМ!</p>
    </div>
  )
}
