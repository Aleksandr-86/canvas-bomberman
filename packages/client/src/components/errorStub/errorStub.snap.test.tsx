import * as renderer from 'react-test-renderer'
import { ErrorStub } from './errorStub'

describe(`<ErrorStub/> snapshot test cases`, () => {
  test(`<ErrorStub/> should render <div className=errorStub> parent with child elements`, () => {
    const tree = renderer.create(<ErrorStub />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
