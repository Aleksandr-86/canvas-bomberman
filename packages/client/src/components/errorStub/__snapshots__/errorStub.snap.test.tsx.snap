// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorStub/> snapshot test cases <ErrorStub/> should render <div className=errorStub> parent with child elements 1`] = `
<div
  className="errorStub"
>
  <p>
    <span
      className="errorStubTextDanger"
    >
      БАААБАААХ!
    </span>
     И ВСЁ СЛОМАЛОСЬ!
  </p>
  <img
    alt="Герой символизирующий ошибку"
    draggable={false}
    height={426}
    width={426}
  />
  <p>
    НО МЫ УЖЕ ЧИНИМ!
  </p>
</div>
`;
